com.microsoft.mrtk.spatialmanipulation/Editor/ObjectManipulator/ObjectManipulatorInspector.cs

```diff
@@ -13,7 +13,7 @@ namespace Microsoft.MixedReality.Toolkit.SpatialManipulation.Editor
     /// </summary>
     [CustomEditor(typeof(ObjectManipulator))]
     [CanEditMultipleObjects]
-    public class ObjectManipulatorInspector : StatefulInteractableInspector
+    public class ObjectManipulatorInspector : StatefulInteractableEditor
```

Comment:
> ObjectManipulator**Inspector** 

https://en.wikipedia.org/wiki/Turtles_all_the_way_down

Reply:
but really, we can do cleanup passes in other packages as we go. doesn't need to all be in this PR!

Reply:
Haven't pushed those changes yet haha sorry!

Reply:
All fixed!